### PR TITLE
Do not assign ColocatedTasks to admins

### PR DESCRIPTION
**Pull request by lowellrex**
Connects #7973. Prevents `ColocatedTask`s from being assigned to admins on the VLJ support team (`Colocated`).

### CODE REVIEW

File: app/models/organization.rb [hunk 1]
```diff
@@ -3,6 +3,14 @@ class Organization < ApplicationRecord
   has_many :organizations_users, dependent: :destroy
   has_many :users, through: :organizations_users
 
+  def admins
+    organizations_users.select(&:admin?).map(&:user)
+  end
+
+  def non_admin_users
+    organizations_users.reject(&:admin?).map(&:user)
```
Comment (lowellrex) [inline app/models/organization.rb]: I played around with using associations to do this in a more rails-y fashion, but couldn't get it to work. Open to ideas for how to make this more rails-y if that's something we want to do.

Comment (amprokop) [reply app/models/organization.rb]: 👍this is good

File: app/models/organization.rb [hunk 2]
```diff
@@ -3,6 +3,14 @@ class Organization < ApplicationRecord
   has_many :organizations_users, dependent: :destroy
   has_many :users, through: :organizations_users
 
+  def admins
```
Comment (amprokop) [inline app/models/organization.rb]: Hm — possible to name these two functions more consistently?

File: app/models/tasks/colocated_task.rb [hunk 3]
```diff
@@ -36,7 +36,7 @@ def create_many_from_params(params_array, user)
     private
 
     def list_of_assignees
-      Colocated.singleton.users.order(:id).pluck(:css_id)
```
Comment (amprokop) [inline app/models/tasks/colocated_task.rb]: any reason we changed `order` to `sort_by`?

File: app/models/organization.rb (hunk 2 above)
Comment (lowellrex) [reply app/models/organization.rb]: I think admins and non_admins makes more sense.

**Pull request merged**